src/query/api/v1/handler/prometheus/native/common.go

```diff
@@ -328,25 +294,14 @@ func RenderResultsJSON(
 			limited = true
 			break
 		}
-		seriesRendered++
-		datapointsRendered += length
-
-		jw.BeginObject()
-		jw.BeginObjectField("metric")
-		jw.BeginObject()
-		for _, t := range s.Tags.Tags {
-			jw.BeginObjectField(string(t.Name))
-			jw.WriteString(string(t.Value))
-		}
-		jw.EndObject()
-
-		jw.BeginObjectField("values")
-		jw.BeginArray()
 
+		currentDatapoint := 0
```

Comment:
nit; can be a bool since value is unused after incrementing once

Reply:
Done